e2e/snapshot_ops_test.go

```diff
@@ -314,6 +314,8 @@ func testRestoredVolumeMount(t *testing.T, tc *testCluster) {
 	err := mountVolume(host, snapTestName, mntPath)
 	r.Nil(err, fmt.Sprintf("mount failed: %s", err))
 
+	defer syscall.Unmount(mntPath, syscall.MNT_FORCE|syscall.MNT_DETACH)
```

Comment:
Please don't use `syscall.MNT_DETACH`. Here and elsewhere in this PR. These are all glusterfs mounts and not XFS mounts.

Remove it from `e2e/glustershd_test.go` too